modules/adding-yaml-examples-to-kube-resources.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * web-console/customizing-the-web-console.adoc
+
+[id="adding-yaml-examples-to-kube-resources_{context}"]
+= Adding YAML examples to Kubernetes resources
+
+You can dynamically add YAML examples to any Kubernetes resources at any time.
+
+.Procedure
+
+. From *Administration* -> *Custom Resource Definitions*, click on *ConsoleYAMLSample*.
+
+. Click the *Create Console Instances Sample* button.
+
+. Edit the YAML file:
++
+----
+apiVersion: console.openshift.io/v1
+kind: ConsoleYAMLSample
+metadata:
+  name: example
+spec:
+  targetResource:
```

Comment:
There's an optional `snippet` property that lets you define a fragment instead of the entire doc. See https://github.com/openshift/api/pull/509